src/providers/v2/subgraph-provider.ts

```diff
@@ -254,6 +254,10 @@ export class V2SubgraphProvider implements IV2SubgraphProvider {
 
     metric.putMetric(`V2SubgraphProvider.chain_${this.chainId}.getPools.filter.latency`, Date.now() - beforeFilter);
     metric.putMetric(`V2SubgraphProvider.chain_${this.chainId}.getPools.filter.length`, poolsSanitized.length);
+    metric.putMetric(
+      `V2SubgraphProvider.chain_${this.chainId}.getPools.filter.percent`,
+      (poolsSanitized.length / pools.length) * 100
```

Comment:
nit: you can specify the metric type as percent

Reply:
thought about it, but I think thats just a presentation thing which we dont really need much right now